client/my-sites/plans-features-main/index.tsx

```diff
@@ -25,9 +22,10 @@ import {
 	ComparisonGrid,
 	PlanTypeSelector,
 	useGridPlans,
-	usePlanFeaturesForGridPlans,
-	useRestructuredPlanFeaturesForComparisonGrid,
+	useGridPlansForFeaturesGrid,
+	useGridPlansForComparisonGrid,
 } from '@automattic/plans-grid-next';
+import useGridPlanForSpotlight from '@automattic/plans-grid-next/src/hooks/data-store/use-grid-plan-for-spotlight';
```

Comment:
```suggestion
import { useGridPlanForSpotlight } from '@automattic/plans-grid-next';
```

Let's export this from the main index and update the import here. Also, going forward, let's avoid direct imports from `/src`

Reply:
Good catch, updated: [8530310](https://github.com/Automattic/wp-calypso/pull/87377/commits/853031081a30ec4ce8fa764dfe2b3202023fee9c)